core/src/display_object/movie_clip.rs

```diff
@@ -1307,7 +1307,7 @@ impl<'gc> MovieClip<'gc> {
             frame_pos = reader.get_ref().as_ptr() as u64 - tag_stream_start;
 
             use swf::TagCode;
-            let tag_callback = |reader: &mut SwfStream<'gc>, tag_code, tag_len| match tag_code {
+            let tag_callback = |reader: &mut _, tag_code, tag_len| match tag_code {
```

Comment:
Ideally, the required change would be to only remove the lifetime annotation:
```diff
-            let tag_callback = |reader: &mut SwfStream<'gc>, tag_code, tag_len| match tag_code {
+           let tag_callback = |reader: &mut SwfStream<'_>, tag_code, tag_len| match tag_code {
```
but this unfortunately exposes another rust bug that requires such a hacky solution. See rust-lang/rust#58052 rust-lang/rust#70263.